python/test/t_Mesh_std.expout

```diff
@@ -10,8 +10,8 @@ volume= 2.2
 First simplex volume= 1.0
 is p= [1.3]  in mesh?  True
 Nearest index( [1.8] )= 1
-Nearest index( [1.8] ), simplex and coordinates= [[1,1], class=Point name=Unnamed dimension=2 values=[0.5,0.5]]
-Nearest index( [-1.8] ), simplex and coordinates= [[0], class=Point name=Unnamed dimension=0 values=[]]
+Nearest index( [1.8] ), simplex and coordinates= [True, 1L, class=Point name=Unnamed dimension=2 values=[0.5,0.5]]
+Nearest index( [-1.8] ), simplex and coordinates= [False, 0L, class=Point name=Unnamed dimension=0 values=[]]
```

Comment:
1L, 0L: You may want to cast to int here this is the formatting of the long type, which was removed in Python3